subsys/net/l2/ethernet/ethernet.c

```diff
@@ -815,6 +815,39 @@ static struct ethernet_vlan *get_vlan(struct ethernet_context *ctx,
 	return NULL;
 }
 
+static void setup_ipv6_link_local_addr(struct net_if *iface)
```

Comment:
shouldn't this function be under ifdef CONFIG_NET_IPV6?

Reply:
no need, it is used from IS_ENABLED() block which is compiled away so this function will disappear too

Reply:
oh yes, true